src/transports/dash/manifest_parser.ts

```diff
@@ -180,9 +181,10 @@ export default function generateManifestParser(
           return processMpdParserResponse(value.continue(resources));
         }
       }).catch((err) => {
-        const error = err instanceof Error
-          ? err
-          : new Error("An unknown error occured when fetching external ressources.");
+        const error = formatError(err, {
```

Comment:
I'm under the impression that this catch here can also be triggered for a regular parsing error (happening in `value.continue`) that may not be linked to the action of fetching the resource.

To me, it makes more sense that the default error thrown here is a parsing error, no?